### ISSUE TITLE
Advene Window Installer not Working

### ISSUE THREAD

**Issue by sanyaade-teachings** [oaubert/advene#91]
Hello Everybody,

I downloaded Windows installer from Advene website, run the installer to set it up, the installation went well and completed without any issue,
I double clicked on the Advene launcher icon but not happen. I went into the Advene bin folder and double clicked on Advene.exe, still nothing happen. 

My Daugther had same issue on her Windows 10 laptop.

Please help!

Many thanks!

God blesses!!!

Best regards,
Sanyaade

**Comment by sanyaade-teachings** [comment]
Hi All,

We managed to get this sorted out. Its the shortcut issues on Windows. you need to got into Advene/bin folder and then double-clicked on Advene.exe to run the application.

I am now looking at how to stack up Advene in docker. I need this!

Many thanks!

God blesses!!!

Best regards,
Sanyaade

**Comment by oaubert** [comment]
Hi. Could you elaborate on this "shortcut issues on Windows"? Is this specific to your Windows setup, or is it more generic? I could have a go at fixing https://github.com/oaubert/advene/blob/master/dev/win_installer/misc/win_installer.nsi#L174 but I would need more information.

**Comment by sanyaade-teachings** [comment]
@oaubert :
I think it is a general issue on Windows. I had the same issue and my daughter had it as well on her laptop.
As for me the shortcut was point to C:\program files\bin\advene.exe but during the installation, I selected a different folder for installation.

My daughter on just accepted the Advene default installation steps but still the shortcut icon not working. I have to show her how to go into the Advene\bin folder to get it running. I thinks it has to do with installer configuration/issues.

NOTE: Also just to bring tom your notice as well that the webserver does not work from inside Advene, the w3 icon is disable. I have been scratch my head on that as of this writing. Looks like Cherrypy webserver not properly packaged with Advene. I went through Advene folder including the python lib/site-packages to see if I could pin down the problem, even to the extend of trying to add cherrypy through the terminal from inside Advene folder (using Advene own packaged Python). I will see how this go as I interested in Advene access over the web to collaborate on with my daughter over distance at her university.

Many thanks for your continuous support and all your efforts on Advene. Much appreciated!

God blesses!!!

Best regards,
Sanyaade
